Fix api-resources table parsing

simonswine opened the pull request:
For sometime `tk pruning` fails, as my cluster has overlapping CRD names/mismatching Kind and Name fields . Errors persisted like this:

```
$ tk prune environments/default
fetching UID's .. done 555.50452ms
fetching previously created resources .. Error: error: the server doesn't have a resource type "NetworkAttachmentDefinition"
```

The reason why this was happing was a failed parsing of the `-o wide` table of `kubectl api-resources`: The field `APIGROUP` no longer exists and is now part of `APIVERSION`.

julienduchesne commented:
@simonswine. I think this could maybe cause issues with people running older kubectl versions. For example,`kubectl` 1.18 will still return the `APIGROUP` column. Could this be done in a way where both are supported? Check APIVERSION and then APIGROUP if it's not set?

simonswine commented:
@julienduchesne very good point. I have fixed this for both versions. 

I also noticed the test data was not fully accurate, so I replaced it with real one that I retrieved locally.

julienduchesne reviewed (approved):
LGTM!
